users/management/commands/unlock_and_generate_backup_code.py

```
@@ -0,0 +1,68 @@
+import secrets
+
+from django.core.management.base import BaseCommand
+
+from users.models import ConnectUser
+
+
+class Command(BaseCommand):
+    help = "Unlock a user and generate a new backup code for them"
+
+    def add_arguments(self, parser):
+        parser.add_argument("--phone-number", type=str, required=False)
+        parser.add_argument("--inactive-user-id", type=str, required=False, help="User ID of inactive user to unlock")
+        parser.add_argument("--disable-current-active-user", action="store_true")
+
+    def handle(self, *args, **options):
+        phone_number = options.get("phone-number")
+        inactive_user_id = options.get("inactive-user-id")
+        if not phone_number and not inactive_user_id:
+            print("Please provide either a phone number or an inactive user ID")
+            return
```

not blocking, but this should probably raise an exception or at least return an non-success status code